Dockerfile

```diff
@@ -11,6 +11,7 @@ MAINTAINER ICGC <dcc-support@icgc.org>
 ENV EGA_VERSION 2.2.2
 ENV GT_VERSION 3.8.7
 ENV GT_VERSION_LONG 207
+ENV GDC_VERSION gdc-client_v1.0.1_Ubuntu14.04_x64
```

Comment:
Well, you would have had to update it in that case as well. Hopefully they will stabilize. 

Btw, is this the latest version?